apps/mobile/src/contexts/ManageWalletContext.tsx

```diff
@@ -99,6 +99,11 @@ const ManageWalletProvider = memo(({ children }: Props) => {
     bottomSheet.current?.dismiss();
   }, []);
 
+  const clearState = useCallback(() => {
```

Comment:
it still seems a bit odd to me that we call `setIsSigningIn(false)` and `provider?.disconnect();` when the app is opened 🤔 

I would expect the opposite where signing in is false by default and provider is disconnected, and they get set to true + connected when the user triggers a sign in or wallet connect flow

Reply:
@kaitoo1 Make sense! I think I need to rename the `isSigningIn` naming.

The current `isSigninIn` state is to describe that the user is in the progress of sign in (not failed or success).